### PR TITLE
Improve handling of status code/message (#51)

### PR DESCRIPTION
**Pull request by owais**
Improved how translators handling mapping status codes and messages to and from OC

translator/trace/README.md documents the new behaviour: https://github.com/open-telemetry/opentelemetry-service/pull/70/files#diff-cf653ae20e304fa43f02c3a37a01a3cd

### CODE REVIEW

File: translator/trace/protospan_translation.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2019, OpenTelemetry Authors
+// Copyright 2018, OpenTelemetry Authors
```
Comment (tigrannajaryan) [inline translator/trace/protospan_translation.go]: Fix here and elsewhere too.

File: translator/trace/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* `message` should be added as a `status.message` tag.
+
+### OC to Zipkin
+
+When converting from OC to Zipkin, if the OC span has the status field set, then
+
+* `code` should be added as a `census.status_code` tag.
+* `message` should be added as a `census.status_description` tag.
+
+In addition to this, if the OC status field represents a non-OK status, then a tag with the key `error` and value `true` should be added to the Zipkin span if one is not already present.
```
Comment (adriancole) [inline translator/trace/README.md]: please do not do this. zipkin has always regarded the "error" true an anti pattern. instrumentation have been known to set false nonsense value

the value of the "error" tag should be the best available message ex status code if a text variant is not available. empty message is ok and better than true.

**Comment by owais** [comment]
Updated.

File: translator/trace/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+* `message` should be added as a `status.message` tag.
+
+### OC to Zipkin
+
+When converting from OC to Zipkin, if the OC span has the status field set, then
+
+* `code` should be added as a `census.status_code` tag.
+* `message` should be added as a `census.status_description` tag.
+
+In addition to this, if the OC status field represents a non-OK status, then a tag with the key `error` should be added and set to the same value as that of the status message falling back to status code when status message is not available.
```
Comment (adriancole) [inline translator/trace/README.md]: SGTM

**Pull request merged**